website/docs/advanced/impression-data.md

```diff
@@ -79,7 +79,7 @@ You can enable impression data via the impression data toggle in the admin UI's
 ## Example setup
 
 :::caution
-This functionality is currently only supported in the [Unleash Proxy client](../sdks/proxy-javascript.md) and [React Proxy client](../sdks/proxy-react.md).
+This functionality is currently only supported in some of our SDKs, please see the [compatibility table for SDKs](../sdks/index.md#server-side-sdk-compatibility-table).
```

Comment:
Same thing as above: I think we should keep mention of the proxy clients in the note.

```suggestion
This functionality is currently only supported the JavaScript-based proxy clients and some of some of the server-side clients. Please refer to the [compatibility table for SDKs](../sdks/index.md#server-side-sdk-compatibility-table).
```

Reply:
Added.